media/css/custom.scss

```diff
@@ -498,3 +498,24 @@ summary {
   height: 40px;
   padding: 8px;
 }
+
+.finance-blocks{
+  display: block;
+}
+
+.finance-block{
+  float: left;
+  width: 228px;
+  margin: 16px;
+  height: 168px;
+  text-align: center;
+}
+
+.finance-block h2{
```

Comment:
Merge rule `.finance-block h2` with rule on line 506<br>Opening curly brace `{` should be preceded by one space